gensrc/proto/internal_service.proto

```diff
@@ -495,6 +495,17 @@ message PMVMaintenanceTaskResult {
     required StatusPB status = 1;
 };
 
+message PExecuteCommandRequest {
```

Comment:
```suggestion
message ExecuteCommandRequestPB {
```

Reply:
all other messages in this file  use Pxxx convention, are you sure that we change to PB suffix?

Reply:
yes, we can change them later